### DISCUSSION TITLE
Ameriflux 2021 Poster Feedback

### DISCUSSION THREAD

**Discussion by AdamTheisen** [ARM-DOE/ACT#336] [General]
There was some good feedback from the Ameriflux 2021 poster session and am hoping for more under this discussion some notable items are

- Using ACT for data (visualization, retrievals) from drone flights which has synergies with ARM's efforts on the UAS/AAF/TBS
- Develop a script for pulling PBL data from NOAA
- Pull in other libraries for more interactive plotting
- Look at ability to bridge python with R
- Develop a toolbox in the documentation of other related software of note (similar to xarray page)

**Comment by AdamTheisen** [comment]
![Screen Shot 2021-06-11 at 1 27 46 PM](https://user-images.githubusercontent.com/13421074/121733243-dc13a400-cab8-11eb-9df0-ee1ee5942c11.png)

Traffic has ramped up leading up to the poster session

**Comment by kenkehoe** [comment]
"Develop a script for pulling PBL data from NOAA". Where can we get this data?

**Comment by AdamTheisen** [comment]
> "Develop a script for pulling PBL data from NOAA". Where can we get this data?

I was not quite sure from the conversation. It sounded like it may be satellite data and in that case I think we want to stay out of that arena. I'll do some digging
